Makefile

```diff
@@ -1436,6 +1436,13 @@ LINT_CPP_FILES = $(filter-out $(LINT_CPP_EXCLUDE), $(wildcard \
 	tools/snapshot/*.h \
 	))
 
+FORMAT_CPP_FILES ?=
+FORMAT_CPP_FILES += $(LINT_CPP_FILES)
+FORMAT_CPP_FILES += $(wildcard \
```

Comment:
It seems somewhere above, the test files are added to LINT_CPP_EXCLUDE which gets filtered out in the final LINT_CPP_FILES? Can we just add LINT_CPP_EXCLUDE back here?

Reply:
I removed these files from the exclusion list and cpplint didn't report any failure. No idea why cpp test files are excluded but it seems fine to add them back.

Updated. Thanks for pointing this out!